lib/cypress_on_rails/middleware.rb

```diff
@@ -16,7 +16,7 @@ def initialize(app, command_executor = CommandExecutor, file = ::File)
 
     def call(env)
       request = Rack::Request.new(env)
-      if request.path.start_with?("#{configuration.api_prefix}/__cypress__/command")
+      if request.path.start_with?("#{configuration.api_prefix}/__e2e__/command")
```

Comment:
another breaking change, we should also check `__cypress__` path and print a deprecation warning.